### PR TITLE
Update setup instructions with notes TAP have made along the way

### PR DESCRIPTION
**Pull request by forceedge01**
## What

TAP has gone through the setup as per the readme and found a few bits that we're missing. We've added these notes in to the readme.

## Checklist

Before you ask people to review this PR:

- Tests and rubocop should be passing: `bundle exec rake`
- Github should not be reporting conflicts; you should have recently run `git rebase main`.
- There should be no unnecessary whitespace changes. These make diffs harder to read and conflicts more likely.
- The PR description should say what you changed and why, with a link to the JIRA story.
- You should have looked at the diff against main and ensured that nothing unexpected is included in your changes.
- You should have checked that the commit messages say why the change was made.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -100,7 +100,9 @@ ALLOW_FUTURE_SUBMISSION_DATE
 ```
 
 Set ALLOW_FUTURE_SUBMISSION_DATE to true to allow integration tests to run with submission dates that are in the future.
-A copy of the `.env` file including the current values can be found in the `Shared-LAA` section of LastPass.
+A copy of the `.env` file including the current values can be found in the `Shared-LAA` section of LastPass (reach out to the team if you don't have access).
+
+Note: Remove “GOOGLE_SHEETS_” from several of the variables in .env file
```
Comment (colinbruce) [inline README.md:105]: Did you download the correct file?  I cannot see any `GOOGLE_SHEETS_` lines in the CFE .env file.
There _are_ some in the *Apply* file though 😕

File: README.md [hunk 2]
```diff
@@ -116,6 +118,11 @@ for testing new thresholds before they come into affect on production.
 
 ## Setup
 
+Please install the following dependencies prior to running the application setup:
+
+brew install shared-mime-info
+brew install postgresql
```
Comment (colinbruce) [inline README.md:124]: Fair point - most developers would have installed these as Apply which is generally the first service we use :+1:

**Review by colinbruce — COMMENTED** [review]
Looks good, but the `GOOGLE_SHEETS_` comment may be on the wrong file

**Comment by naseberry** [comment]
Author left org.
PR replaced by https://github.com/ministryofjustice/check-financial-eligibility/pull/1233

**Pull request closed without merging**